platforms/ide/problems-api/src/integTest/groovy/org/gradle/api/problems/ProblemsServiceIntegrationTest.groovy

```diff
@@ -38,10 +39,12 @@ class ProblemsServiceIntegrationTest extends AbstractIntegrationSpec {
         given:
         withReportProblemTask """
             problems.forNamespace('org.example.plugin').reporting {
-                it.details('Wrong API usage')
+                it.details('Wrong API usage, will not show up anywhere')
```

Comment:
I was using this test to try things and I didn't realize that it won't be part of the problem report, because the wrong api usage causes a distinct problem.
I'm hoping to make sure I remind that in the future.